tests/pybullet_helpers/robots/test_panda.py

```diff
@@ -0,0 +1,124 @@
+"""Tests for PandaPyBullet Robot."""
```

Comment:
we've run into trouble in the past when adding `__init__.py` files inside the test directory. is it definitely needed here? if the checks pass, it's fine though

Reply:
I don't think it's strictly needed, I found a long time ago (maybe it's patched now) that PyCharm sometimes doesn't find tests if there isn't the `__init__.py`.

Also, if you have test utility modules this allows you to discover them. For now, we don't need so I will see if we can remove.

I'm curious what were the past troubles?

Reply:
I think it was essentially this: https://stackoverflow.com/questions/41748464/pytest-cannot-import-module-while-python-can